packages/manager/apps/pci-public-ip/src/pages/order/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,15 @@
+import { cloneElement } from 'react';
+
+export default function Stepper({
+  children,
+}: {
+  children: JSX.Element[];
+}): JSX.Element {
+  return (
+    <div className="flex flex-col gap-y-4 mt-4">
+      {children.map((child: JSX.Element, i: number) =>
+        cloneElement(child, { key: i, order: i + 1 }),
```

Comment:
why do you need to clone it ?

Reply:
to avoid passing the order as prop, this way we may have two times the same order, or even worse, order 2 that preceds order 1